src/librbd/Journal.cc

```diff
@@ -7,6 +7,7 @@
 #include "common/Timer.h"
 #include "common/WorkQueue.h"
 #include "cls/journal/cls_journal_types.h"
+#include "journal/FutureImpl.h"
```

Comment:
Why does the private implementation header file need to be pulled in to all external client translation units?

Reply:
That was an accident, I'll remove those lines.

Reply:
Done.